.github/workflows/desktop.yml

```diff
@@ -50,44 +39,7 @@ jobs:
         run: |
           echo "::set-env name=VCPKG_RESPONSE_FILE::external/vcpkg_${{ matrix.vcpkg_triplet }}_response_file.txt"
           echo "::set-env name=PIP_REQUIREMENTS_FILE::external/pip_requirements.txt"
```

Comment:
is PIP_REQUIREMENTS_FILE still required?

Reply:
Nope, it wasn't. Thanks for catching it. Have removed it now.